src/lib/builders/accordion/create.ts

```diff
@@ -130,7 +137,7 @@ export const createAccordion = <T extends AccordionType = 'single'>(
 						return $value === itemValue ? undefined : itemValue;
 					});
 				}),
-				addEventListener(node, 'keydown', (e) => {
+				addMeltEventListener(node, 'keydown', (e) => {
```

Comment:
We should probably add a Space keydown handler, as cancelling the click makes space not work anymore. Sometimes you may want to only disable clicks (IDK why anyone would do that, butttt)